### PR TITLE
Fix create-project

### PR DESCRIPTION
**Pull request by deizel**
Composer was failing due to: branch change @ friendsofcake/crud; and gourmet/email#3.

Also, updated the entire app template because `post-install-cmd` was failing:

```
[InvalidArgumentException] Maximum number of attempts must be a positive value.
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -153,7 +153,7 @@ Copyright (c) 2015, Jad Bitar and licensed under [The MIT License][mit].
 [milestones]://github.com/gourmet/platform/issues/milestones
 [migrations/repo]://github.com/cakephp/migrations
 [mit]:http://www.opensource.org/licenses/mit-license.php
-[mobiledetect/repo]://github.com/mobiledetectlib/mobiledetectlib
+[mobiledetect/repo]://github.com/serbanghita/Mobile-Detect
```
Comment (jadb) [inline README.md:156]: Why?

https://github.com/cakephp/app/blob/master/composer.json#L10

File: composer.json [hunk 2]
```diff
@@ -30,7 +30,7 @@
         "gourmet/faker": "~1.0",
         "gourmet/robo": "dev-master",
         "friendsofcake/bootstrap-ui": "*",
-        "friendsofcake/crud": "dev-cake3",
+        "friendsofcake/crud": "dev-master",
```
Comment (jadb) [inline composer.json:33]: Maybe better `~4.0`?

File: config/routes.php [hunk 3]
```diff
@@ -5,23 +5,32 @@
 
 Router::defaultRouteClass('Route');
 
-Router::scope('/', function($routes) {
+Router::scope('/', function ($routes) {
```
Comment (jadb) [inline config/routes.php:8]: cs no?

File: phpunit.xml.dist [hunk 4]
```diff
@@ -1,33 +1,33 @@
...
+		<listener
+		class="\Cake\TestSuite\Fixture\FixtureInjector"
+		file="./vendor/cakephp/cakephp/src/TestSuite/Fixture/FixtureInjector.php">
+			<arguments>
+				<object class="\Cake\TestSuite\Fixture\FixtureManager" />
+			</arguments>
+		</listener>
+	</listeners>
 
 </phpunit>
```
Comment (jadb) [inline phpunit.xml.dist:33]: Why spacing changed here?

File: README.md (hunk 1 above)
Comment (deizel) [inline README.md:156]: Visit http://github.com/mobiledetectlib/mobiledetectlib in your browser and you will see :wink:

File: config/routes.php (hunk 3 above)
Comment (deizel) [inline config/routes.php:8]: https://github.com/cakephp/app/commit/e05e9b595729d21ef5b26399cd029e9a7f3560cf

File: phpunit.xml.dist (hunk 4 above)
Comment (deizel) [inline phpunit.xml.dist:33]: Hmm.. I just overwrote it with cakephp/app's file.. assumed it was changed to spaces but turns out it's tabs.

File: composer.json (hunk 2 above)
Comment (deizel) [inline composer.json:33]: The last tag was a few months ago (approaching 100 commits) so I left it tracking the dev branch.

Let me know if you want to track tagged versions instead and I'll make the change.

**Comment by jadb** [comment]
Thanks!

**Comment by deizel** [comment]
No problem :)

**Pull request merged**